resources/build/bootstrap-ubuntu-20.04.sh

```diff
@@ -22,8 +22,8 @@ sudo apt-get install -y --no-install-recommends libfontenc-dev libice-dev libsm-
 
 # Install additional build tools.
 sudo pip3 install -r "$WORKSPACE/resources/build/requirements.txt"
-# Use explicit predictable conan root path, to be used for both packages
-# and conan CMake toolchain config.
+# Use explicit predictable conan root path, where packages are 
+# installed.
 export CONAN_USER_HOME="$HOME/conan"
```

Comment:
[question] No we're no longer installing into this, does it still make sense to set this? If I understand it right, this is just going to make the conan cache end up in ` ~/conan/.conan` instead of `~/.conan`?  Given we have to hard-code that path in the CI cache step anyway, not sure it makes much difference?

Reply:
Fine to keep it - but would it be worth setting to `~` or similar instead so we're not adding a random dir? Is this something we should just bring into the workspace too, given we're actively managing its population with a cache stage?

Reply:
According to the [docs](https://docs.conan.io/en/latest/reference/env_vars.html#conan-user-home), `CONAN_USER_HOME` only talks about the cache dir - so we can assume its always inside there?

Reply:
I'd certainly be happy to remove it though if you didn't see the specific merit, as it makes it easier to grok what's happening.

Reply:
> then decided there was no point in complicating the PR with that change and so reverted it?

Yeah, this is my recollection, we reasoned the existing setup is equivilant to `~` in all but style, so why bother adding the noise

Reply:
Do you mean why bother removing it? The existing setup explicitly _isnt_ the same as `~`, as it moves the cache from `~/.conan` to `~/conan/.conan`.

Reply:
An other potential reason to remove it - this bootstrap script is used by Vagrant and as 'suggested' instructions for people. By setting a custom `CONAN_USER_HOME` you then have to remember that this was set before, if you want to interact with conan in future, unrelated commands as the env var may no longer be set.

If we leave it out, then you can use conan 'as normal' in subsequent commands, and not have to remember that it was moved before. So you don't end up with two separate conan folders from differing `install` commands.

Reply:
Either way though, lets save this for a follow up PR as we'd need to update the CI cache etc...

Reply:
> you then have to remember that this was set before, if you want to interact with conan in future

That's a pretty convincing argument, shall I make an issue?